src/libs/api.js

```diff
@@ -0,0 +1,4 @@
+export default endpoint => {
+	const api = location.hostname === 'github.com' ? 'https://api.github.com/' : `${location.origin}/api/`;
+	return fetch(api + endpoint).then(res => res.json());
+};
```

Comment:
This file looks unused?

Reply:
Currently, yes, but it's not being included at the moment. I feel it's a start for whenever the API materializes.